plugins/engines/ospray/OSPRayEngine.cpp

```diff
@@ -204,7 +206,9 @@ Vector2ui OSPRayEngine::getSupportedFrameSize(const Vector2ui& size)
         return Engine::getSupportedFrameSize(size);
 
     Vector2f result = size;
-    if (getCamera().getType() == CameraType::stereo)
+    const auto& rp = _parametersManager.getRenderingParameters();
```

Comment:
camera still has stereo mode, why use rendering parameters here?

Reply:
As discussed, the stereo mode is now at an "application" level rather than camera because of the impacts on the deflect streaming.